ideasbox/templatetags/ideasbox_tags.py

```diff
@@ -21,3 +24,10 @@ def theme_slug(inst, slug=None):
         slug = SLUGS.get(name, name.lower())
     klass = THEMES.get(name, slug)
     return mark_safe(tpl.format(klass=klass, slug=slug))
+
+
+@register.filter()
+def remove_i18n(url):
+    """Remove i18n prefix from an URL."""
+    return i18n_pattern.sub("/", url)
+i18n_pattern = re.compile(language_code_prefix_re)
```

Comment:
My PEP8 sense tells me to always put 2 blank lines between any module-level declarations (classes, functions, variables).